smoketest/scripts/cli/base_vyostest_shim.py

```diff
@@ -89,11 +86,7 @@ def cli_commit(self):
             # during a commit there is a process opening commit_lock, and run() returns 0
             while run(f'sudo lsof -nP {commit_lock}') == 0:
                 sleep(0.250)
-            # wait for FRR reload to be complete
-            while process_named_running('frr-reload.py'):
-                sleep(0.250)
-            # reset getFRRconfig() guard timer
-            self.commit_guard = time()
+            sleep(5)
```

Comment:
Can we make it a named constant and provide some considerations about the value? My immediate thought is "can we make it shorter?" or, in general, "can we reduce the wait time by doing a more precise condition check?". Let's document that knowledge.

Reply:
Good one!

Added proper code documentation:
```
    The commit process is not finished until all pending files from
    VYATTA_CHANGES_ONLY_DIR are copied to VYATTA_ACTIVE_CONFIGURATION_DIR. This is
    done inside libvyatta-cfg1 and the FUSE UnionFS part. On large non-interactive
    commits FUSE UnionFS might not replicate the real state in time, leading to
    errors when querying the working and effective configuration.

    TO BE DELETED AFTER SWITCH TO IN MEMORY CONFIG
```